main.go

```diff
@@ -50,13 +50,13 @@ func main() {
 			<head><title>Hasura Exporter</title></head>
 			<body>
 			<h1>Hasura Exporter</h1>
-			<p><a href="` + settings.WebAddr + `/metrics">Metrics</a></p>
+			<p><a href="` + settings.Port + `/metrics">Metrics</a></p>
 			</body>
 			</html>`))
 	})
 
-	logrus.Info("started hasura exporter on ", settings.WebAddr)
+	logrus.Info("started hasura exporter on ", settings.Port)
 
-	logrus.WithField("msg", http.ListenAndServe(settings.WebAddr, nil)).Fatalln()
```

Comment:
`WebAddr` is in this format in case you wanna bind the exporter to a specific interface such as `196.188.12.61:9921`, it's not just a port it can also include an ip address to bind to